### ISSUE TITLE
issue with child components

### ISSUE THREAD

**Issue by appsdevpk** [jegan321/nuro#14]
there is an issue with child components, when i include the todos-list component as shown in the code below then the if directive works only first time, but when i remove this component from app template everything works fine, one more thing, if i include this child component at the beginning (before h1) then all the reactivity stops working.

```
class App {
			apptitle = 'A Test App';
			count = 0;
			name = 'world';
			template = `
				<div :id="props.id" :data-count="count">
					<h1>{{apptitle}}</h1>
					<button @click="increment">Clicked {{count}} times</button><br />
					<input type="number" $bind="count"/><br />
					Name: <input $bind="name"/> <br />
					{{props.greeting}}, {{name}}!<br />
					<div $if="count > 5">asdfasdfasdf</div>
					<div id="todoslist">
						<todos-list title="Todos List"></todos-list>
					</div>
				</div>
			`;
			increment(){
				this.count++;
			}
		}
		Nuro.include('TodosList',class{
			tasks = ['First', 'Second', 'Third'];
			template = `
				<div>
					<h1>{{props.title}}</h1>
					<ul>
						<li $for="task in tasks"><todos-item>{{task}}</todos-item></li>
					</ul>
				</div>
			`;
			includes = {TodosItem};
		});
		class TodosItem{
			template = `<h3><slot></slot></h3>`;
		}
		let app = Nuro.mount(App,document.getElementById('appcontainer'),{greeting: 'Hi', id:'myapp'});
```

**Comment by jegan321** [comment]
Hmm, definitely a bug. It looks like the slot content is not being passed to the child component after the update for some reason. I will look at it some more tomorrow probably. Thanks for testing out the library!

**Comment by jegan321** [comment]
Hi @appsdevpk, this bug is now fixed in the main branch and v0.5.2. Great catch!
